import.py

```diff
@@ -649,7 +652,63 @@ def write_to_file(cont: str, filename: str) -> None:
         f.write(cont)
 
 
+def prepend_file(filename: str, new_str: str) -> None:
+    with open(filename, "r") as original_f:
+        with open(filename + "_", "w") as new_f:
+            new_f.write(new_str)
+            new_f.write(original_f.read())
+    os.rename(filename + "_", filename)
```

Comment:
Simpler to skip the renaming step and instead first read and then write. We don't need atomicity here.

Also, I'd inline this at the usage because it's such a specific thing to be doing and so short

Reply:
okay done